addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1553,7 +1553,15 @@ export class Rtc {
             return;
         }
         const oldCount = channel.rtcSessions.length;
+        const hadSelfSession = Boolean(this.state.selfSession?.in(channel.rtcSessions));
         channel.rtcSessions = sessionsData;
+        if (
+            !hadSelfSession ||
+            hadSelfSession !== Boolean(this.state.selfSession?.in(channel.rtcSessions)) ||
+            !this.store.env.services["multi_tab"].isOnMainTab()
+        ) {
+            return;
+        }
```

Comment:
In other words: non-main-tab and non-self self session transitions do not produce any sound